### PR TITLE
tweaks: allow dev copy of compose & set permissions for container

### PR DESCRIPTION
**Pull request by zehicle**
The directory create will require a rebuild but is not urgent so we can wait until the next rebuild.

### CODE REVIEW

File: containers/rebar-api/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM digitalrebar/deploy-service-wrapper
 ENTRYPOINT ["/sbin/docker-entrypoint.sh"]
 
 RUN groupadd -r rebar && useradd -r -u 1000 -g rebar -p '$6$afAL.34B$T2WR6zycEe2q3DktVtbH2orOroblhR6uCdo5n3jxLsm47PBm9lwygTbv3AjcmGDnvlh0y83u2yprET8g9/mve.' -m -d /home/rebar -s /bin/bash rebar \
-  && mkdir -p /opt/digitalrebar /etc/sudoers.d /home/rebar/.ssh /var/run/rebar /var/cache/rebar/ansible_playbook \
+  && mkdir -p /opt/digitalrebar /etc/sudoers.d /home/rebar/.ssh /var/run/rebar /var/cache/rebar/ansible_playbook /home/rebar/.ansible \
   && chown rebar:rebar /var/run/rebar
```
Comment (galthaus) [inline containers/rebar-api/Dockerfile]: Did you want to set the permissions?  You just created it, I think.  You would need something like the chown or chmod line to make permissions/ownership changes?

Comment (zehicle) [inline containers/rebar-api/Dockerfile]: I thought that these were being created w/ rebar:rebar.  I can add a chown too.

Comment (galthaus) [inline containers/rebar-api/Dockerfile]: They all aren't.  /opt/digitalrebar gets fixed on container start.  I suspect the last four need chmod, /home/rebar/.ssh /var/run/rebar /var/cache/rebar/ansible_playbook /home/rebar/.ansible.

Comment (zehicle) [inline containers/rebar-api/Dockerfile]: ok, I can address this.

**Comment by zehicle** [comment]
merging w/ another request....

**Pull request closed without merging**